Gruntfile.js

```diff
@@ -269,6 +282,16 @@ module.exports = function(grunt) {
                       'concat',
                       'uglify']);
 
+  grunt.registerTask('update-webdriver', function () {
```

Comment:
Looking into enhancing [grunt-protractor-runner](https://github.com/teerapap/grunt-protractor-runner) so that it will automatically do this on install or with an option in the task.